php-resource/package.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "sample-php-resource",
+    "type": "module",
```

Comment:
Generally, packages that are not supposed to be published should have `"private": "true"` - not sure if this is the case across samples repos but I figure it's worth mentioning at least